crates.io release is broken

Adding this crate to a project that's only a week old immediately produced the following:

```
error[E0432]: unresolved import `log::LogRecord`
 --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:2:22
  |
2 | use log::{self, Log, LogRecord, LogLocation, LogLevel, LogLevelFilter, SetLoggerError};
  |                      ^^^^^^^^^ no `LogRecord` in the root. Did you mean to use `Record`?

error[E0432]: unresolved import `log::LogLocation`
 --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:2:33
  |
2 | use log::{self, Log, LogRecord, LogLocation, LogLevel, LogLevelFilter, SetLoggerError};
  |                                 ^^^^^^^^^^^ no `LogLocation` in the root

error[E0432]: unresolved import `log::LogLevel`
 --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:2:46
  |
2 | use log::{self, Log, LogRecord, LogLocation, LogLevel, LogLevelFilter, SetLoggerError};
  |                                              ^^^^^^^^ no `LogLevel` in the root

error[E0432]: unresolved import `log::LogLevelFilter`
 --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:2:56
  |
2 | use log::{self, Log, LogRecord, LogLocation, LogLevel, LogLevelFilter, SetLoggerError};
  |                                                        ^^^^^^^^^^^^^^ no `LogLevelFilter` in the root?

error[E0412]: cannot find type `LogMetadata` in module `log`
  --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:75:40
   |
75 |     fn enabled(&self, _metadata: &log::LogMetadata) -> bool {
   |                                        ^^^^^^^^^^^ did you mean `Metadata`?

error[E0046]: not all trait items implemented, missing: `flush`
  --> /home/ralith/.cargo/registry/src/github.com-1ecc6299db9ec823/systemd-0.1.0/src/journal.rs:74:1
   |
74 | / impl Log for JournalLog {
75 | |     fn enabled(&self, _metadata: &log::LogMetadata) -> bool {
76 | |         true
77 | |     }
...  |
81 | |     }
82 | | }
   | |_^ missing `flush` in implementation
   |
   = note: `flush` from trait: `fn(&Self)`

error: aborting due to 6 previous errors

error: Could not compile `systemd`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,10 +13,10 @@ include = ["Cargo.toml", "**/*.rs", "README.md" ]
 bus = ["libsystemd-sys/bus"]
 
 [dependencies]
-log = "0.*"
-libc = "0.*"
-utf8-cstr = "0.*"
-mbox = "0.*"
+log = "~0.3"
+libc = "~0.2"
+utf8-cstr = "~0.1"
+mbox = "~0.4"
 
 [dependencies.libsystemd-sys]
 path = "libsystemd-sys"

